.github/workflows/linux.yml

```diff
@@ -25,7 +25,7 @@ env:
   SCCACHE_CACHE_SIZE: 30G
   SCCACHE_AZURE_KEY_PREFIX: genai/ubuntu/22_04/x64
   HF_HOME: /mount/caches/huggingface/lin
-  OV_CACHE: /mount/caches/huggingface/.ov_cache/lin
+  OV_CACHE: /mount/caches/huggingface/.ov_cache/lin/187f4d4
```

Comment:
why changed?

Reply:
This is test purpose, since I found that transformers version used for model export are different as python test, would like to regenerate OV cache to check if which version is more stable for CI test.